net-misc/seafile-client/seafile-client-6.0.4-r1.ebuild

```diff
@@ -43,6 +45,7 @@ src_configure() {
 	export QT_SELECT=qt5
```

Comment:
uhm definitely not. You must use `eqmake5` or fix the build system to detect the correct Qt version. Most cmake-based packages that support both Qt4 and Qt5 expose an option to select between the two.